mem/ctrl/CtrlMemRTL.py

```diff
@@ -72,7 +73,224 @@ def update_raddr():
           else:
             s.reg_file.raddr[0] <<= s.reg_file.raddr[0] + AddrType( 1 )
 
-  def line_trace( s ):
-    out_str  = "||".join([ str(data) for data in s.reg_file.regs ])
-    return f'{s.recv_ctrl.msg} : [{out_str}] : {s.send_ctrl.msg}'
+  # verbose trace if verbosity > 0
```

Comment:
Is that possible to extract some logic from the huge verbose_trace() func into the lib folder, which can then be used by other modules? It looks super huge now.

Reply:
Yes, think the same thing when I wrote them.

Reply:
done